reproducible/ubuntu/nodebootstrap.bzl

```diff
@@ -56,7 +56,7 @@ docker rm $cid
 nodebootstrap = rule(
     attrs = {
         "_builder_image": attr.label(
-            default = Label("//reproducible/ubuntu:nodejs_builder.tar"),
+            default = Label("//reproducible/ubuntu:node_8_5_0_builder.tar"),
```

Comment:
Why is this one hardcoded? Should we leave off a default to prevent people from making mistakes?